src/chatlog/documentcache.h

```diff
@@ -22,11 +22,12 @@
 #include <QStack>
 
 class QTextDocument;
+class SmileyPack;
 
 class DocumentCache
 {
 public:
-    DocumentCache() = default;
+    DocumentCache(SmileyPack& smileyPack);
```

Comment:
explicit? Having a SmileyPack being implicitly convertible to a DocumentCache doesn't feel good. :P

Reply:
Ah! Thanks, fixed.